### ISSUE TITLE
Could not find module 'Html'

### ISSUE THREAD

**Issue by jmatsushita** [circuithub/elm-bootstrap-html#1]
Hi there,

I get the following error after:
- Creating an empty project directory
- Running `elm-package install circuithub/elm-bootstrap-html`
- Then running `elm-make`

```
Error when searching for modules imported by module 'Bootstrap.Html':
    Could not find module 'Html'

Potential problems could be:
  * Misspelled the module name
  * Need to add a source directory or new dependency to elm-package.json
```

Sorry if its obvious as I'm an elm beginner!

Best,

Jun

**Comment by cutsea110** [comment]
Hi jmatsuhita.

I guess elm-make couldn't build elm-bootstrap-html.
Because the package dependencies doesn't have elm-html package,
while the Bootstrap.Html module import Html(this module require elm-html).
You try to modilfy elm-stuff/packages/circuithub/elm-bootstrap-html/4.0.0/elm-package.json like this.
( add dependencies evancz/elm-html package)

{
    "version": "4.0.0",
    "summary": "Html shorthand for Bootstrap",
    "repository": "https://github.com/circuithub/elm-bootstrap-html.git",
    "license": "MIT",
    "source-directories": [
        "src"
    ],
    "exposed-modules": [
        "Bootstrap.Html"
    ],
    "dependencies": {
        "circuithub/elm-html-shorthand": "4.1.0 <= v < 5.0.0",
        "evancz/elm-html": "2.0.0 <= v < 3.0.0",
        "elm-lang/core": "1.0.0 <= v < 2.0.0"
    }
}

**Comment by rehno-lindeque** [comment]
Oh geez, sorry I see you did start an issue here. I don't know why I didn't get the notification... I assumed that `elm-html` would be pulled in `with elm-html-shorthand`? Am I missing something?

**Comment by jmatsushita** [comment]
Thanks @cutsea110 your fix works. @rehno-lindeque Would be great to update the package with this explicit dependency to avoid others running into the problem until there's clarity on why this happens in the first place!

**Comment by rehno-lindeque** [comment]
Sorry for not doing it earlier. Hope that sorts you out! Updated dependencies is published at version [4.0.2](http://package.elm-lang.org/packages/circuithub/elm-bootstrap-html/4.0.2/)

**Comment by rehno-lindeque** [comment]
PS, please be aware the next version of bootstrap-html will end up looking a little bit differently in terms of the conventions.
